spec/policies/admin_only_user_profile_policy_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe AdminOnly::UserProfilePolicy do
+
+  let(:user_1) { create(:user, email: 'user@random.com') }
+  let(:member) { create(:member_with_membership_app, email: 'member@random.com', company_number: '5562728336')}
+  let(:admin)  { create(:user, email: 'admin@shf.com', admin: true) }
+  let(:visitor) { build(:visitor) }
+
+  CRUD_ACTIONS = [:edit, :update, :become]
```

Comment:
"become" is not  a CRUD action.  doesn't belong here.

Reply:
Done.